addons/website_event/views/event_event_views.xml

```diff
@@ -34,15 +34,15 @@
                 <field name="is_published" widget="website_redirect_button"/>
             </div>
             <xpath expr="//div[hasclass('oe_title')]" position="after">
-                <div name="event_menu_configuration" groups="base.group_no_one">
-                    <label for="website_menu" string="Website Submenu"/>
+                <div name="event_menu_configuration" groups="base.group_no_one" class="mb-3">
                     <field name="website_menu"/>
+                    <label name="website_menu" for="website_menu" string="Website Submenu"/>
                     <!-- hidden sub-menus, they are triggered all at once based on "website_menu" -->
                     <field name="introduction_menu" invisible="1"/>
                     <field name="location_menu" invisible="1"/>
                     <field name="register_menu" invisible="1"/>
+                    <field class="ms-5" name="community_menu" invisible="1"/>
```

Comment:
To keep the checkboxes and their corresponding labels together irrespective of screen size, wrapped each pair in a `div` element and applied necessary CSS classes. Check it out once and see if it's fine by you.

P.S: In case of `event_type_views`, they were wrapped in a `span` but it doesn't really work in this case. Hence I've now changed them to `div` too.